src/__tests__/index.js

```diff
@@ -1,15 +1,25 @@
 /* eslint func-style:0 */
 import React from 'react'
 import * as glamor from 'glamor'
-import {render} from 'enzyme'
+import {render, mount} from 'enzyme'
 import * as jestGlamorReact from 'jest-glamor-react'
 import {oneLine} from 'common-tags'
+import toJson from 'enzyme-to-json'
```

Comment:
Because we use [the serializer](https://github.com/paypal/glamorous/blob/14970c1bb59d1408d3066d209dafa356b5ce3173/package.json#L97-L99) this shouldn't be necessary :tada: